### PR TITLE
Remove internationalization beta

### PR DESCRIPTION
**Pull request by roryabraham**
### Details
The locale picker cannot be used from the signup or login pages because betas are not yet available (because the user is not logged in). Dropped a message in Slack [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1630450923241100), but I feel like at this point we can safely remove this beta entirely.

### Fixed Issues
$ https://github.com/Expensify/App/issues/4958

**Note:** The above issue is not reproducible on dev, because all betas are enabled by default.

### Tests / QA Steps
1. Go to signin page.
1. There should be a locale picker in the bottom of the signin page. Changing it should change the language.
1. Go to the password page.
1. There should be a locale picker in the bottom of the password page. Changing it should change the language.

### Screenshots
Not including screenshots because it looks the same on dev as it did before – this change will only be visible on staging.

### CODE REVIEW

**Comment by roryabraham** [comment]
cc @iwiznia Because internationalization was originally your project I believe.

**Review by aldo-expensify — DISMISSED** [review]
The changes look good to me. Tested in web and Android.

We may want to create a follow up GH issue to keep the chosen language once we login or signup.

File: src/libs/Permissions.js [hunk 1]
```diff
@@ -51,19 +51,10 @@ function canUseDefaultRooms(betas) {
     return _.contains(betas, CONST.BETAS.DEFAULT_ROOMS) || canUseAllBetas(betas);
 }
 
-/**
- * @param {Array<String>} betas
- * @returns {Boolean}
- */
-function canUseInternationalization(betas) {
-    return _.contains(betas, CONST.BETAS.INTERNATIONALIZATION) || canUseAllBetas(betas);
```
Comment (Jag96) [inline src/libs/Permissions.js:59]: Can you also remove this beta from const.js if it is unused now?

**Comment by roryabraham** [comment]
> We may want to create a follow up GH issue to keep the chosen language once we login or signup.

Reported in #expensify-open-source 👍

File: src/libs/Permissions.js (hunk 1 above)
Comment (roryabraham) [reply src/libs/Permissions.js:59]: Updated

**Review by Jag96 — APPROVED** [review]

**Comment by roryabraham** [comment]
Merging to resolve deploy blocker

**Comment by iwiznia** [comment]
Oh crap! I was OOO and did not catch this. We should not remove the beta, the internationalization project is still one, given we only have one language and that we have not internationalized the server (and thus a lot of the messaging people see). Remember we only added internationalization early in the App so that it would be easier to internationalize later.
Can we revert this and re-add the beta please?

**Comment by Jag96** [comment]
@iwiznia thanks for the context, I'll put up a revert PR for this

**Pull request merged**